### PR TITLE
Nemo 5627/enhance our homepage

### PR DESCRIPTION
**Pull request by mindyschwab**
### Ticket(s)

- _[Airtable ticket](https://airtable.com/appfJZShN8K4tcWHU/pagxblGyhI5EJIA3v?HjEAY=recO9LJZuWYhn6Be6)_

### Type of change

- [X] Design change

### Description

Include a summary of the change or which issue was is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

1. On both mobile and desktop, updated “Anchor link text button group DK” from 3 buttons to 4 buttons. The new button is titled “1st link” with the text “OUR MISSION’”. It links to “CARD/ Our Mission” on homepage.   

2. On both mobile and desktop, added new “our mission” card to the top of the homepage. Card is positioned at X=200 and Y= 220 on desktop, and X=16 and Y=0 on mobile.  

3. On desktop and mobile, we’ve changed “Card/ How it works” to include 4 steps instead of 3. The fourth and final step is new. It is titled “Right Side”. It is made up of Frame 479, and includes the image titled “Chart_Two Color-2 1” and text block “We help you explore what comes after determining your vacation eligibility.” 
Additionally, remove the titles "inform","assess" and "calculate from the card.

4. Update the card "Why Vacate" on desktop to be a single line of four items, instead of two lines of two items each.

Figma: https://www.figma.com/file/icusYj7kJl2xmYkudIZEG4/CV-Current-Design-Files?type=design&mode=design&t=ag08AMi9CJkkmOak-0_

### Screenshots
Screen Shot 2023-07-31 at 6 36 13 PM

Screen Shot 2023-07-31 at 6 36 19 PM

Screen Shot 2023-07-31 at 6 36 26 PM

#### After: 
Screen Shot 2023-07-31 at 6 37 51 PM

### Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings in the console
- [x] There are no new linting errors/problems in my code
- [x] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

File: content/home.ts [hunk 1]
```diff
@@ -31,28 +39,32 @@ const content: HomeContent = {
...
+        },
       ],
     },
     {
       id: 'why-vacate',
       title: 'Why Vacate?',
-      subtitle: 'A conviction vacation seals the offense from your record, and will give you more chance to access resources like:',
+      ctaLink: '/get-started',
+      ctaText: 'Get started',
+      subtitle: 'A conviction vacation seals the offense from your record, and will give you more chance to access:',
```
Comment (e-papanicolas) [inline content/home.ts:67]: Can you please update the subtitle to: "A conviction vacation seals the offense from your record, giving you a better chance at access to:". I made a comment in the Figma file to have it changed there as well. It doesn't sound right to me.

File: pages/index.tsx [hunk 2]
```diff
@@ -62,10 +66,40 @@ export default function Home() {
...
+                  src={content.ourMission.imgsrc as string}
+                  alt=""
+                  width={406}
+                  height={306}
+                  style={{ width: '100%' }}
+                  useImageDimensions
+                />
+              </Grid>
+              <Grid item sm={12} md={6}>
+                <Typography variant="subtitle1" sx={{ mt: '4.375rem', textAlign: 'center' }}>
```
Comment (e-papanicolas) [inline pages/index.tsx:88]: Text align here should only be center for xs and sm, on desktops it should be left aligned. See `HeroBanner.tsx` line 99 for an example of how to use different sizes within `sx`.

File: pages/index.tsx [hunk 3]
```diff
@@ -62,10 +66,40 @@ export default function Home() {
...
+            </Grid>
+          </Box>
+        </SectionContainer>
+
         {content.gridSections.map((section) => (
           <SectionContainer key={section.id} id={section.id}>
             <PaperSection
               title={section.title}
+              subtitle={section.subtitle}
+              // added subtitle above
```
Comment (e-papanicolas) [inline pages/index.tsx:102]: Let's try and keep comments only for explaining something that might be difficult to understand. When we review the pull requests, everything you have added, changed, or deleted is highlighted so we don't need to add these.

**Comment by e-papanicolas** [comment]
Mindy, great work! I have left a few comments in the code requesting a few small changes. 

I also noticed that the FAQ on the Figma is a new one, so it needs to be added to the content file.

Also a tip on filling out the PR template. If you click create without marking any of the boxes, on the next screen they become checkboxes, MUCH easier to use. Also, you should delete all of the unused checkboxes so that on the main pull requests screen it shows all tasks complete for the PR.

File: components/layout/SectionContainer.tsx [hunk 4]
```diff
@@ -15,6 +15,7 @@ export default function SectionContainer({
       {...props}
     >
       {children}
+
```
[the remark this answers is not in the document]
Comment (mindyschwab) [reply components/layout/SectionContainer.tsx:18]: Line was removed

File: content/home.ts [hunk 5]
```diff
@@ -18,10 +19,17 @@ const content: HomeContent = {
...
   sectionNavs: [
+    { label: 'Our Mission', href: '#our-mission' },
     { label: 'How it works', href: '#how-it-works' },
     { label: 'Why vacate', href: '#why-vacate' },
     { label: 'FAQ', href: '#faq' },
   ],
+  ourMission: {
+    header:
+      'Our Mission',
+    body: 'Clearviction was founded in 2020 sparked by [Washington State’s New Hope Act](https://app.leg.wa.gov/billsummary?BillNumber=2890&Year=2017). We are a 501(c)(3) non-profit organization committed to helping people with convictions improve their quality of life.\n\n\n\n\n\nEach year, nearly 98k people are added to the millions across Washington State already burdened with a criminal record. The justice-impacted folks will face significant barriers to their day to day life after completing their prison terms.',
```
[the remark this answers is not in the document]
Comment (mindyschwab) [reply content/home.ts:30]: Nope, was just trying something out.  Extra \n removed

File: content/home.ts (hunk 1 above)
Comment (mindyschwab) [reply content/home.ts:67]: updated

File: pages/index.tsx [hunk 6]
```diff
@@ -62,10 +66,40 @@ export default function Home() {
             ))}
           </ButtonGroup>
         </SectionContainer>
+        <SectionContainer sx={{ margin: 'auto', maxWidth: '65rem' }} maxWidth={false}>
+          <Box>
```
[the remark this answers is not in the document]
Comment (mindyschwab) [reply pages/index.tsx:70]: replaced Box with PaperSection

File: pages/index.tsx (hunk 3 above)
Comment (mindyschwab) [reply pages/index.tsx:102]: Removed comment

File: pages/index.tsx (hunk 2 above)
Comment (mindyschwab) [reply pages/index.tsx]: Updated

File: content/home.ts [hunk 7]
```diff
@@ -117,6 +129,11 @@ const content: HomeContent = {
       summary: 'Does vacating a conviction make it disappear from my record?',
       details: 'No.\n\nYour conviction will still be in court records and computerized court indexes to court records. If a conviction was a domestic violence case, these records and indexes will still show the case type.\n\nInformation about the court records from the cases that led to the convictions are still public. You can still find them on [www.courts.wa.gov](https://www.courts.wa.gov/)\n\nProsecutors can still use evidence of vacated convictions in a later criminal prosecution. They can still use them in a sexually violent predator commitment proceeding.\n\nFBI records and private background check records may still have information about the convictions.',
     },
+    {
```
Comment (e-papanicolas) [inline content/home.ts:132]: Will you move this FAQ to the top of the list? On the design it looks like it should be first. I think you can bypass changing all of the ids for now by making it faq0.

File: pages/index.tsx [hunk 8]
```diff
@@ -62,10 +66,48 @@ export default function Home() {
             ))}
           </ButtonGroup>
         </SectionContainer>
+        <SectionContainer sx={{ margin: 'auto', maxWidth: '65rem' }} maxWidth={false}>
```
Comment (e-papanicolas) [inline pages/index.tsx:69]: I think that we might want to remove the maxWidth here, it looks like it is causing this paper section to be a different size than the rest on desktop screens.

**Review by e-papanicolas — CHANGES_REQUESTED** [review]
Mindy, just a few small tweaks and then this PR will be ready to go, thanks for taking care of all of the comments.

File: content/home.ts (hunk 7 above)
Comment (mindyschwab) [reply content/home.ts:132]: Updated

File: pages/index.tsx (hunk 8 above)
Comment (mindyschwab) [reply pages/index.tsx:68]: I see what you mean - I removed the maxWidth and margin properties

**Review by e-papanicolas — APPROVED** [review]

**Pull request merged**